src/lib/components/workflow/workflows-summary-configurable-table/tests/__snapshots__/table-body-cell.test.ts.snap

```diff
@@ -12,7 +12,7 @@ exports[`Table_body_cell > History Size renders 1`] = `"<td class="workflows-sum
 
 exports[`Table_body_cell > Parent Namespace renders 1`] = `"<td class="workflows-summary-table-body-cell svelte-qupenl" data-testid="workflows-summary-table-body-cell">parent-namespace-id</td>"`;
 
-exports[`Table_body_cell > Run ID renders 1`] = `"<td class="workflows-summary-table-body-cell filterable svelte-qupenl" data-testid="workflows-summary-table-body-cell"><a href="/namespaces/default/workflows/abc-123/def-456/history" class="link inline svelte-1xo1ry6">  def-456</a> </td>"`;
+exports[`Table_body_cell > Run ID renders 1`] = `"<td class="workflows-summary-table-body-cell filterable svelte-qupenl" data-testid="workflows-summary-table-body-cell"><a href="/namespaces/default/workflows/abc-123/def-456/history" class="link inline svelte-1m1kigg">  def-456</a> </td>"`;
```

Comment:
I support just getting rid of these snapshots. They're more trouble than their worth.

Reply:
I am 100% down to delete them 🗑️ cc @rossedfort

Reply:
Yep, delete em

Reply:
[Deleted](https://github.com/temporalio/ui/pull/2118/commits/4f48e6a403f13c5c085d01758723afefab8399f6) ✅